src/main/scala/org/zouzias/spark/lucenerdd/query/LuceneQueryHelpers.scala

```diff
@@ -107,6 +107,13 @@ object LuceneQueryHelpers extends Serializable {
     indexSearcher.search(q, topK).scoreDocs.map(SparkScoreDoc(indexSearcher, _))
   }
 
+  def searchParser(indexSearcher: IndexSearcher,
+                   query: Query,
+                   topK: Int, analyzer: Analyzer)
```

Comment:
Analyzer should be removed here, since analysis happened during the definition of `Query`/